Hotfix/study detail

yesjjin99 opened the pull request:
- 비로그인 상태에서 스터디 개별 페이지 접근 시 서버에 문제가 생기는 문제가 아직도 존재하는 것 같아서 해당 부분을 조금 수정했습니다
   - 우려되는 부분이 해당 로직 중간중간 존재하는 catch 문에서도 200을 응답해주고 있는데, checkToken에서는 401을 응답해주는 부분으로 로그아웃 상태임을 나타내주기 위해 isLogIn 이라는 값을 함께 반환하도록 추가해주었습니다. 이에 대해 프론트에서도 반영이 필요할 듯 싶습니다ㅠㅠ

yesjjin99 commented:
- 스터디에서 제목과 내용을 500자까지 작성 가능하도록 해야 한다는 이슈가 있어 변경했습니다
- 스터디 삭제도 제대로 동작이 안된다고 해서 `onDelete: CASCADE` 옵션 추가해주었습니다

Jeyoung-Park commented on src/entity/NoticeEntity.ts:
```diff
@@ -13,10 +13,10 @@ export default class Notice {
   id!: string;
 
-  @Column({ name: 'TITLE' })
+  @Column('varchar', { name: 'TITLE', length: 500 })
```
타이틀은 40자입니다!

rudy3091 commented on src/routes/study/study.controller.ts:
```diff
@@ -237,51 +240,90 @@ const getStudybyId = async (req: Request, res: Response) => {
       if (!accessToken && refreshToken) {
-        await refresh(req, res);
-
-        if (!req.cookies.accessToken) return;
```
앗 지금보니 여기가 pending 되는 문제의 지점이었을 수도 있겠네요
서버를 재시작하고 액세스토큰이 만료되었는데 어떠한 이유로 refresh 함수 처리 과정에서 액세스토큰이 재발급되지 않아서 응답을 내려주지 않는..?
이번 PR 로 해결될 수 있었으면 좋겠네요 🥲

yesjjin99 committed:
Fix study detail logic to access without login

Add length option to title, about column

Add onDelete: CASCADE option to study related entities
